pkg/virt-handler/vm.go

```diff
@@ -618,6 +619,7 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 
 			// If any of domain.Status.Interfaces were not handled above, it means that the vm contains additional
 			// interfaces not defined in domain.Spec (most likely added by user on VM). Add them to vmi.Status.Interfaces
```

Comment:
Just a nit - I would also update the comment "(most likely added by user on VM or SRIOV interface)"

Reply:
But they are in the Spec, `domain.Spec.Devices.HostDevices`
This comment is mostly for the pure manual added interfaces i think, if for example the user consoled

Reply:
ok, so s/domain.Spec/domain.Spec.Devices.Interfaces. The comment would be more accurate if it will include the sriov interfaces as well.